packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -0,0 +1,235 @@
+                padding: localizeSpacing(Direction.ltr)(`${toPx(12)} ${toPx(30)} ${toPx(12)} ${toPx(12)}`),
+            },
+            "& a": {
+                textAlign: "left",
+                display: "block",
+                padding: localizeSpacing(Direction.ltr)(`${toPx(12)} ${toPx(30)} ${toPx(12)} ${toPx(12)}`),
+                "& span": {
+                    display: "block",
+                    fontStyle: "italic",
+                    fontSize: toPx(13),
+                    paddingTop: toPx(4)
+                }
+            }
+        }
+    };
+}
+
+export function applyAriaHiddenStyles(): ICSSRules<{}> {
+    return {
+        "&[aria-hidden='true']": {
+            display: "none"
+        },
+        "&[aria-hidden='false']": {
+            display: "block"
+        }
+    };
+}
+
+export function applyPopupMenuStyles(): ICSSRules<{}> {
+    return {
+        position: "absolute",
+        right: "0",
+        width: toPx(200),
+        background: colors.menuGray,
+        zIndex: "1",
+        top: toPx(55),
+        "& li > button": {
+            cursor: "pointer",
+            display: "block",
+            padding: localizeSpacing(Direction.ltr)(`${toPx(12)} ${toPx(12)} ${toPx(12)} ${toPx(40)}`),
+            position: "relative",
+            border: "none",
+            background: "transparent",
+            textAlign: "left",
+            "&:before": {
+                position: "absolute",
+                content: "''",
+                opacity: ".6",
+                pointerEvents: "none",
+                top: toPx(12),
+                width: toPx(16),
+                height: toPx(16),
+                left: toPx(12),
+                /* tslint:disable-next-line */
+                background: "url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTMiIGhlaWdodD0iMTYiIHZpZXdCb3g9IjAgMCAxMyAxNiIgdmVyc2lvbj0iMS4xIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIj48ZyBpZD0iQ2FudmFzIiBmaWxsPSJub25lIj48ZyBpZD0iJiMyMzg7JiMxNTk7JiMxMzE7Ij48cGF0aCBkPSJNIDE0IDQuMjg5MDZMIDE0IDE2TCAxIDE2TCAxIDBMIDkuNzEwOTQgMEwgMTQgNC4yODkwNlpNIDEwIDRMIDEyLjI4OTEgNEwgMTAgMS43MTA5NEwgMTAgNFpNIDEzIDE1TCAxMyA1TCA5IDVMIDkgMUwgMiAxTCAyIDE1TCAxMyAxNVoiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC0xIDApIiBmaWxsPSJibGFjayIvPjwvZz48L2c+PC9zdmc+) center no-repeat"
+            },
+            "&:focus": {
+                outline: "none"
+            }
+        }
+    };
+}
+
+export function applyPopupHeadingStyles(): ICSSRules<{}> {
+    return {
+        position: "relative",
+        "& > div": {
+            display: "flex",
+            borderBottom: `${toPx(1)} solid ${colors.border}`,
+            "& h3": {
+                flexGrow: "1",
+                "& + button": {
+                    fontSize: toPx(14),
+                    background: "none",
+                    border: "none",
+                    padding: `${toPx(2)} ${toPx(12)}`,
+                    color: "#0078D7",
```

Comment:
same as other comment, can we use colors.blue here?

Reply:
Fixed.